src/main/java/gov/usgs/cida/pubs/transform/CrossrefTransformer.java

```diff
@@ -188,7 +224,11 @@ protected void writeComment(String message) throws IOException {
 		//add error message as a comment to the xml document
 		bufferedWriter.append(wrapInComment(message));
 	}
-	
+
+	protected String getPubIdMsg(Publication<?> pub) {
```

Comment:
I'm not a fan of all these one line methods, makes it harder to determine what is really going on.